src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -80,6 +74,17 @@
         return currentBiomes;
     }
 
+    private static List<OceanBiomeEntry>[] setupOceanBiomes()
```

Comment:
EnumMap would be a much cleaner solution here, no?

Reply:
In my opinion, it would be a much cleaner solution, yes. 

The thought of having it be an `EnumMap` did cross my mind when I was first making the change to the field, but it would be inconsistent with the already present `biomes` field in the class, so I decided not to do it and stay consistent.